base/logging.go

```diff
@@ -550,15 +550,17 @@ func NewLoggerWriter(logKey string, serialNumber uint64, req *http.Request) *Log
 
 func CreateRollingLogger(logConfig *LogAppenderConfig) {
 	if logConfig != nil {
+		SetLogLevel(logConfig.LogLevel.sgLevel())
+
 		lj := lumberjack.Logger{}
 
-		if logConfig.LogFilePath != nil {
+		if logConfig.LogFilePath == nil {
```

Comment:
I found this logic a little bit hard to follow at first read (we create the lumberjack logger, THEN abort when LogFilePath == nil - it took me a minute to work out whether we were using the rolling logger or not)

What do you think about doing the "if LogFilePath==nil" check right after SetLogLevel, and then just fall through to the rest?